src/simtools/db/db_handler.py

```diff
@@ -24,20 +25,47 @@
 # The above comment is because pylint does not know that DatabaseHandler.db_client is subscriptable
 
 
+jsonschema_db_dict = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema#",
+    "type": "object",
+    "description": "MongoDB configuration",
+    "properties": {
+        "db_server": {"type": "string", "description": "DB server address"},
+        "db_api_port": {
```

Comment:
Should we add a default here as well? We usually only use 27017.

Reply:
Done.